files/69-available-updates-begin.sh.amzn2

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/bash
+
+# Copyright (C) 2024 Amazon.com, Inc. or its affiliates.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#    http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+# OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the
+# License.
+
+package_updates=$(LANG=C timeout 30s /usr/bin/yum \
+  --debuglevel 2 \
+  --security check-update 2>/dev/null \
+  | grep -P '(?<! 0 packages) available$')
+
+if [ -n "$package_updates" ]; then
+  echo "$package_updates
+Update to the latest ECS-Optimized AMI for all updates, including security"
+
+  security_updates=$(grep -oP '(\d+)(?=.* security)' <<< "$package_updates")
+
+  if [ ${security_updates:=0} -ne 0 ]; then
+    echo 'Run "sudo yum update --security" to apply security updates in place'
+  fi
+
+  echo 'Run "sudo yum update" to apply all updates in place'
+fi
+
+# Disable 70-available-updates during update-motd, reverted with
```

Comment:
nit(non-blocking): Could we add a bit more information why we're doing this somewhere either through comments or in the PR description? IIUC, we want these custom messages to show instead of whatever is in `70-available-updates` (please correct me if I'm wrong!). Plus, the ordering is determined by the number prefix for these files right?